lstchain/calib/camera/calibrator.py

```diff
@@ -100,13 +108,31 @@ def __init__(self, subarray, **kwargs):
             )
 
         # declare time calibrator if correction file exist
+
         if os.path.exists(self.time_calibration_path):
+
             self.time_corrector = PulseTimeCorrection(
                 calib_file_path=self.time_calibration_path
             )
         else:
             raise IOError(f"Time calibration file {self.time_calibration_path} not found!")
 
+        # declare the charge sampling corrector
+        if self.time_sampling_correction_path is not None:
+            # search the file in resources if not found
+            if not os.path.exists(self.time_sampling_correction_path):
```

Comment:
If someone specifies a file which is then not found, an error should be raised.

Silently using a default file if a user provided config value is invalid will result in hard to debug issues.

Reply:
Taken out